pkg/grizzly/grafana.go

```diff
@@ -21,17 +22,80 @@ type Folder struct {
 // Board enscapsulates a dashboard for upload to Grafana API
 type Board struct {
 	Dashboard map[string]interface{} `json:"dashboard"`
-	FolderID  int                    `json:"folderId"`
 	Overwrite bool                   `json:"overwrite"`
-	UID       string
-	Name      string
```

Comment:
don't we want to keep `name`? e.g. for `list`?

Reply:
We maintain that information as the key in the `map[string]Board`. For commands like `list` this is sufficient